packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -417,6 +388,16 @@ Generated by [AVA](https://ava.li).
         headers: {},␊
         proxy: false␊
       },␊
+      {␊
+        from: '/external/path',␊
+        path: '/external/path',␊
+        query: {},␊
+        to: '/external/path',␊
+        force: false,␊
+        conditions: {},␊
+        headers: {},␊
+        proxy: false␊
+      },␊
```

Comment:
This is expected.
This is because `@netlify/config` was previously loading redirects after `_redirects` had been removed, which lead to some missing information. This PR fixes this.